staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -59,6 +59,12 @@ const (
 	// Enables compression of REST responses (GET and LIST only)
 	APIResponseCompression featuregate.Feature = "APIResponseCompression"
 
+	// owner: @nabokihms
+	// alpha: v1.25
+	//
+	// Enables API to get self subject attributes after authentication.
+	APISelfSubjectReview featuregate.Feature = "APISelfSubjectReview"
```

Comment:
all uses of this feature are in k8s.io/kubernetes, not in the generic API server k8s.io/apiserver library, so move this to pkg/features/kube_features.go

Reply:
Yeah, I moved it to the `pkg/features`.